### DISCUSSION TITLE
AIR cache on Windows ?

### DISCUSSION THREAD

**Discussion by deesnay** [airsdk/Adobe-Runtime-Support#3122] [Q&A] [answered]
Adobe AIR seems to have a cache somewhere, but I can't find it, on Windows 10.
I'm trying to test file download progress from an AIR application, but as soon as I download once, the subsequent downloads of the same file happen instantly, so I think it must be cached somewhere.
It doesn't seem to be in the same place where the SharedObjects are stored.
Does anyone know?

**Comment by MalacTheLittle** [comment, marked as answer]
It's cached in Edge cache. You can avoid caching by setting `URLRequest.useCache` to `false`.

**Reply by deesnay** [reply]
Thank you! URLRequest.useCache=false does the job, indeed.
However, clearing Microsoft Edge's cache doesn't seem to have any effect...? (Clear Browsing Data > Time range: All time + Cached images and files)

**Comment by MalacTheLittle** [comment]
AIR uses native OS browser cache so not sure how it's not cleared through Edge's Clear Browsing Data, but the other way is to access cache folder from Internet Options > General (tab) > Settings > Temporary Internet Files (tab) > View Files.

![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/56675407/109e3b29-4fb8-4bd7-b5fd-e41de08ebd8d)

**Reply by ajwfrost** [reply]
I think this is the more likely way to clear it ... AIR uses the wininet functions, so it's more similar to the older options that you see above (that you used to access via Internet Explorer).

Edge being Chromium-based I suspect has a cache area that's completely separate from anything related to the rest of the Windows system.
